docs/src/components/docs/breadcrumbs.svelte

```diff
@@ -0,0 +1,22 @@
+<script>
+  import { Button } from 'attractions';
+  import { HomeIcon } from 'svelte-feather-icons';
+
+  export let segment;
+
+  function capitalize(word) {
+    return word[0].toUpperCase() + word.slice(1);
+  }
+</script>
+
+<nav class="breadcrumbs">
```

Comment:
Wouldn't this make for a good component of its own? Perhaps with an array of segments for a prop

Reply:
Maybe, but it would need pretty serious reconstruction, and as it currently stands, it's not meant for general usage